docs/basics/networkobject.md

```diff
@@ -105,7 +105,7 @@ If the player already had a Prefab instance assigned, then the client owns the N
 
 ## Finding PlayerObjects
 
-To find a pPlayerObjects for a specific client ID, you can use the following methods:
+To find a PlayerObjects for a specific client ID, you can use the following methods:
```

Comment:
Ok, I'm going to be that girl...

This isn't your typo, but there's a second typo here: "To find a [singular] PlayerObjects [plural]"

Reply:
Thank you!